.env

```diff
@@ -1,2 +0,0 @@
-API_URL=https://www.api-staging.bimdata.io
-CONNECT_URL=https://connect-staging.bimdata.io
```

Comment:
Why deleting the env variables ?

Reply:
Because we use the prod url by default. It is no longer a 'variable'.
And to be consistent with the code bloc, there is no more env variables in the code.